docs/EMBEDDING.md

```diff
@@ -67,6 +67,8 @@ query string. Currently the following options are available:
 * `logging` - The console log level. Can be one of `error`, `warn`, `info` or
   `debug`.
 
+* `wsProtocols` - WebSocket sub-protocols
```

Comment:
`EMBEDDING.md` is for `vnc.html`, which I can't see any changes for? So either omit this part, or make sure `vnc.html` can pass it on to the `RFB` constructor.